tests/clients_last_seen_v1/test_single_day/expect.ndjson

```diff
@@ -0,0 +1,3 @@
+{"submission_date":"2019-01-02","active_hours_sum":0.0,"attribution":{"source":"prev"},"client_id":"a","last_seen_date":"2019-01-01"}
+{"submission_date":"2019-01-02","active_hours_sum":1.0,"attribution":{"source":"test"},"client_id":"b","last_seen_date":"2019-01-02"}
+{"submission_date":"2019-01-02","active_hours_sum":1.0,"attribution":{"source":"test"},"client_id":"c","last_seen_date":"2019-01-02"}
```

Comment:
we can expect in `ndjson`, `json` or `yaml` format right now. I'm somewhat inclined to switch to `yaml` for date support, but the `bq load` can use this if we keep it as `ndjson` 🤷‍♀️

Reply:
I vote for sticking with what `bq` supports here, too. Coercing between dates and strings is simple enough and I don't anticipate other type compatibility problems.